### DISCUSSION TITLE
为啥n-grid没有虚拟列表？

### DISCUSSION THREAD

**Discussion by ShangChien** [tusen-ai/naive-ui#3149] [Q&A] [answered]
我对grid的理解是，它可以加载许多同一类的item(如n-card, n-thing)。这样的话，n-grid具有虚拟列表特性，体验应该会更棒！

**Comment by 07akioni** [comment, marked as answer]
因为没打算做，这个未来也不会做的

grid 的很多功能强依赖 DOM 信息，这些功能和虚拟列表是冲突的

**Reply by 07akioni** [reply]
虚拟列表的事情让虚拟列表的库来做

**Reply by ShangChien** [reply]
谢谢答疑！
